docs/Users/Install.rst

```diff
@@ -153,6 +153,11 @@ If you want to install coala to an alternate location you can e.g. call
 options are documented on
 https://docs.python.org/3.3/install/#alternate-installation.
 
+..note::
+
+    Make sure you do not use any proxy at installation
```

Comment:
the solution would be telling people about http and https proxy

Sincerely,

Lasse Schuirmann

contact@viperdev.io
http://viperdev.io/

2016-02-26 19:48 GMT+01:00 Sudheesh Singanamalla <notifications@github.com>:

> In docs/Users/Install.rst
> <https://github.com/coala-analyzer/coala/pull/1719#discussion_r54284181>:
>
> > @@ -153,6 +153,11 @@ If you want to install coala to an alternate location you can e.g. call
> >  options are documented on
> >  https://docs.python.org/3.3/install/#alternate-installation.
> >
> > +..note::
> > +
> > +    Make sure you do not use any proxy at installation
>
> I've set this up behind a proxy, I think setting the pip proxy or
> exporting http_proxy and https_proxy should solve the problem. I don't
> think putting a note not to install behind a proxy is the solution to that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1719/files#r54284181>.
>